roles/rke/tasks/common.yaml

```diff
@@ -101,5 +101,5 @@
   wait_for:
     path: /etc/rancher/node/password
     state: present
-    timeout: 180
+    timeout: 1800
```

Comment:
Half an hour for the password seems a fairly long time. Does it take that long?

Reply:
It did on JS1 on TACC, this is an old change